### PR TITLE
Monitoring Microsoft SQL Server 2017

### PR DESCRIPTION
**Pull request by RazeemM**

### CODE REVIEW

File: templates/microsoft-SQLServer2017/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Microsoft SQL Server 2017 Database Monitoring
+=============================================
+
+Monitors the Microsoft SQL Server Database installed on a Windows server.
+
+Provides monitoring of:
+* Logs
+* Data Directory
+* Events
+* Performance metrics.
```
Comment (bertysentry) [inline templates/microsoft-SQLServer2017/README.md]: There's an unwanted dot on this line :-)

File: templates/microsoft-SQLServer2017/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* Running Processes
+* Services
+
+
+In addition, you can add SQL queries and process the results using the "Analysis" feature.
+
+* Import the cfg and activate it on the Host where the SQL Server Database is installed.
+* Set the System Credentials to access the server
```
Comment (bertysentry) [inline templates/microsoft-SQLServer2017/README.md]: System Credentials and sq_sa Credentials are all part of the Host Settings, which is described on line 21. Please rework this part. Also, is it sq_sa or sql_sa ?

File: templates/microsoft-SQLServer2017/README.md (hunk 1 above)
Comment (RazeemM) [reply templates/microsoft-SQLServer2017/README.md]: Dot removed

File: templates/microsoft-SQLServer2017/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* Running Processes
+* Services
+
+
+In addition, you can add SQL queries and process the results using the "Analysis" feature.
+
+Import the cfg and activate it on the Host where the SQL Server Database is installed.
+
+On the Host Settings
+* Provide appropriate Database name and port number under SQLServer Database Connection Settings.
```
Comment (bertysentry) [inline templates/microsoft-SQLServer2017/README.md]: Unwanted dot... :-D

**Comment by bertysentry** [comment]
@Raphyyyy Are you suggesting that we renamed the template itself (its directory and its .CFG file name) to something like "mssql-win"? ("ms" instead of "microsoft" to keep it short). Then later, we can release "mssql-lin". 
@RazeemM Do you feel like it?

**Comment by RazeemM** [comment]
It is mentioned in the readme and description that it for Windows. I thought that should be enough.

**Comment by Raphyyyy** [comment]
Why are you using underscore and sometimes not?
image

**Comment by bertysentry** [comment]
Sure, but for consistency, it's better to have "mssql-win.cfg" and later "mssql-lin.cfg", rather than "mssql.cfg" and "mssql-lin.cfg". Because then, people will be unsure of what platform mssql.cfg is targetting. Yeah, well, it's not very likely. But I find it clearer and more consistent.

**Comment by Raphyyyy** [comment]
> It is men



> 
> 
> @bertysentry @Raphyyyy It is mentioned in the readme and description that it for Windows. I thought that should be enough.

I wasn't thinking about the cfg's name.
I mean we could add "Microsoft SQL Server 2017 (Windows OS)" for example. 

But if we rename the cfg file maybe we don't need to add the OS in the template name :)

**Comment by RazeemM** [comment]
- Updated README
- Removed "_" in the display name
- Changed teplate ID

File: templates/microsoft-SQLServer2017/README.md (hunk 3 above)
Comment (RazeemM) [reply templates/microsoft-SQLServer2017/README.md]: Corrected

**Pull request merged**